diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/NewlinesRule.kt

```diff
@@ -429,7 +437,9 @@ class NewlinesRule(private val configRules: List<RulesConfig>) : Rule("newlines"
      *
      * @return true - if there is error, false, if there is no error and null if there is two calls in chain
```

Comment:
```suggestion
     * @return true - if there is error, false, if there is no error
```
It doesn't return `null` anymore, does it?